src/Media/Media.js

```diff
@@ -0,0 +1,12 @@
+import React, {Component} from 'react';
+
+export default class Media extends Component {
```

Comment:
yup. I also think Media.Header probably should be a seperate component, otherwise we will have folks complaining about the `<h4>` component hard code